src/Microsoft.AspNetCore.Razor.Evolution/RazorProject.cs

```diff
@@ -123,5 +123,15 @@ protected virtual string NormalizeAndEnsureValidPath(string path)
 
             return path;
         }
+
+        /// <summary>
+        /// Create a Razor project based on a physical file system.
+        /// </summary>
+        /// <param name="rootDirectoryPath">The directory to root the file system at.</param>
+        /// <returns></returns>
```

Comment:
@Yves57 if you're around, could you fix this to say `The <see cref="RazorProject" />`. I can touch this up prior to merging it if I don't hear from you.